build.ini

```diff
@@ -60,6 +60,10 @@ deps313 =
     django==4.0.4
     djangorestframework>=3.13,<3.14
 
+deps314 =
+    django==4.1.0
```

Comment:
Is this the lowest version supported by DRF 3.15?

Reply:
I suppose we should use either lowest or highest at the time Django versions for each DRF version...